tip-wlan/charts/opensync-gw-cloud/resources/config/logback.xml

```diff
@@ -73,6 +73,7 @@
 
   <root level="WARN">
     <appender-ref ref="logfile"/>
+    <appender-ref ref="stdout"/>
```

Comment:
I think this should be set to `logfile` as default since we have quite a few tweaks/added functionality (encoders, rollingPolicy etc) to make it easy to maintain. Should be set to `stdout` only for special cases (to look at the logs during startup). Why are both enabled?

Reply:
actually you are right. we should just drop the logfile appender, it doesnt make any sense in k8s environment

Reply:
@rahulsharma2020 we usually suggest that all microservices will send logs to stdout by default. In Kubernetes, this is considered best practice because then you can use cluster-wide log shipping, basically mimicking the suggestions of 12-factor apps (https://12factor.net/logs).
If you feel that's a much more significant change to how we handle things at the moment, we can schedule a call to discuss all the implications of this change.

Reply:
I think there are at least 2 reasons why we need stdout:
1. simplify deployment and maintenance - logs (at least ERROR) should be available via `kubectl logs ..` commands, anyone who troubleshoots pod start failures will appreciate this.
2. local logging is an antipattern in cloud/k8s alike environments where pods come and go.. We'd need to send them to some central location, attach a sidecar to gather these logs - a lot of extra steps instead of just sending to stdout.

Reply:
Thanks @leonidlm, @eugenetaranov-opsfleet. Agree with your explanation.

My take on this: Yes, we would definitely need to look at `stdout` during **startup** - as I mentioned in my above comment and is very useful. However, once the pod is up and running, there would be lots of logs and looking at them using `kubectl logs` isn't easy - especially for services like opensync-gw which spits out tons of logs from APs management. And these logs could easily be uncontrollable if the number of APs managed go in the range of hundreds/thousands. With `logfile` approach, we can limit size of each, copy them on our system and look at them individually and chronologically to check for errors from buggy APs.
Until we have ES/Kibana running to take care of logging from multiple different services, we can go either way.

As originally intended in the PR, let's keep both for now and we can check and decide later if we want to remove anything.

Reply:
thanks for explanation @rahulsharma2020 !
I am not sure if there is a way to disable logs after startup ..
what we might consider later - send ERROR logs to stdout - so they would be useful to maintain the services, and send all logs to some collector forwarding them to elasticsearch ..

Reply:
Agree @eugenetaranov-opsfleet